src/main/java/hudson/plugins/analysis/util/model/AbstractAnnotation.java

```diff
@@ -350,7 +352,7 @@ public final String getFileName() {
     }
 
     @Override
-    public String getTempName(final AbstractBuild<?, ?> owner) {
+    public String getTempName(final Run<?, ?> owner) {
```

Comment:
breaks strict binary compatibility... but maintaining it here would result in having to apply lots of casts if passing `null` parameters...